src/cloudwatch.ts

```diff
@@ -1,30 +1,27 @@
-import { ClientConfiguration as cwConfig, PutMetricDataInput } from "aws-sdk/clients/cloudwatch";
-import { CloudWatch } from "aws-sdk";
+import CloudWatch, { ClientConfiguration as cwConfig, PutMetricDataInput } from "aws-sdk/clients/cloudwatch";
 import { DateTime } from "luxon";
 import { cwLogger } from "./handler";
 import { CloudWatchLogsLogEvent } from "aws-lambda";
-import AWSXRay from "aws-xray-sdk";
+import AWSXRay, { Subsegment } from "aws-xray-sdk";
 import RE2 = require("re2");
 
 export class CW {
-  private readonly config: cwConfig;
+  private readonly config: cwConfig = { region: "eu-west-1", retryDelayOptions: { base: 500 } };
   private readonly branch: string = (process.env.BRANCH ?? "local").toLocaleLowerCase();
```

Comment:
BRANCH env var is expected to be set, this is to avoid a compilation error.

Reply:
I leave it to you but I thought it would easier for the dev if "local" was set as a default branch in case no BRANCH is passed from node (can't see a script in the package.json to run the service which would help any maintainer)

Reply:
This is defaulting to local, if BRANCH env var is null it will be "local"